worlds/ff4fe/__init__.py

```diff
@@ -0,0 +1,397 @@
+# world/mygame/__init__.py
+import argparse
+import json
+import os
+import pkgutil
+import threading
+from typing import Mapping, Any
+
+import Utils
+import settings
+import typing
+
+from .FreeEnterpriseForAP.FreeEnt.cmd_make import MakeCommand
+from .options import FF4FEOptions  # the options we defined earlier
+from .items import FF4FEItem, all_items, ItemData # data used below to add items to the World
+from .locations import FF4FELocation, all_locations, LocationData  # same as above
+from . import topology, flags
+from .itempool import create_itempool
+from . import events, items
+from . import rules as FERules
+from .Client import FF4FEClient
+from worlds.AutoWorld import World, WebWorld
+from BaseClasses import Region, Location, Entrance, Item, ItemClassification, MultiWorld, LocationProgressType
+from .rom import FF4FEProcedurePatch
+from ..generic.Rules import set_rule, add_rule, forbid_item, add_item_rule
```

Comment:
```suggestion
from worlds.generic.Rules import set_rule, add_rule, forbid_item, add_item_rule
```
Need to use absolute imports for things outside of your apworld